audit/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+
+from .models import AuditLog
+
+
+class AuditLogSerializer(serializers.ModelSerializer):
+
+    action_by_name = serializers.CharField(
+        source='action_by.username', required=False)
+    action_name = serializers.SerializerMethodField(required=False)
+
+    class Meta:
+        model = AuditLog
+        read_only_fields = ('action_at', 'action_by_name', 'action_name')
+        fields = ('action_at', 'action_by', 'action', 'model', 'identity_id',
+                  'subscription_id', 'detail', 'action_by_name', 'action_name')
```

Comment:
I think either here in the API, or on the client side, we need to have the action mapping expanded, so that we're not doing a post request with `{"action": "c"}`, which can be confusing in the client code.